### ISSUE TITLE
Console errors : Cannot read property 'modules' / 'name' of undefined

### ISSUE THREAD

**Issue by Lucas-C** [voyages-sncf-technologies/hesperides-gui#70]
When opening hesperides, the following 2 errors keep appearing in the JS console:
```
angular.js:12783 TypeError: Cannot read property 'modules' of undefined
    at Object.platformLocalChanges (localChangesServices.js:82)
    at Object.platformLocalChangesCount (localChangesServices.js:87)
    at Scope.$scope.platformLocalChangesCount (properties.js:1013)
    at fn0 (eval at compile (angular.js:13628), <anonymous>:4:508)
    at expressionInputWatch (angular.js:14650)
    at Scope.$digest (angular.js:16194)
    at Scope.$apply (angular.js:16467)
    at done (angular.js:10852)
    at completeRequest (angular.js:11050)
    at XMLHttpRequest.requestLoaded (angular.js:10991)
angular.js:12783 TypeError: Cannot read property 'name' of undefined
    at Scope.$scope.hasLocalChanges (properties.js:1054)
    at fn (eval at compile (angular.js:13628), <anonymous>:4:341)
    at Scope.$digest (angular.js:16194)
    at Scope.$apply (angular.js:16467)
    at done (angular.js:10852)
    at completeRequest (angular.js:11050)
    at XMLHttpRequest.requestLoaded (angular.js:10991)
```

It looks those errors are also triggered when I hover on a platform.

**Comment by JordanKergoat** [comment]
This issue have already been fixed ba62de3015a8a0bc7125ec01b127604c457239c7

**Comment by Lucas-C** [comment]
Great !
